frontend/src/metabase/dashboard/hooks/use-location-sync.ts

```diff
@@ -5,41 +5,53 @@ import { usePrevious } from "react-use";
 import { omit } from "underscore";
 
 import { DEFAULT_EMBED_DISPLAY_OPTIONS } from "metabase/dashboard/constants";
-import type { DashboardUrlHashOptions } from "metabase/dashboard/types";
 import { parseHashOptions, stringifyHashOptions } from "metabase/lib/browser";
 import { useDispatch } from "metabase/lib/redux";
 import { isNullOrUndefined } from "metabase/lib/types";
 
-const DEFAULT_DASHBOARD_EMBED_DISPLAY_OPTIONS: DashboardUrlHashOptions = {
-  ...DEFAULT_EMBED_DISPLAY_OPTIONS,
+type SYNCED_KEY = "refresh" | "fullscreen" | "theme";
+
+const DEFAULT_DASHBOARD_EMBED_DISPLAY_OPTIONS = {
+  theme: DEFAULT_EMBED_DISPLAY_OPTIONS.theme,
```

Comment:
This tightens the type. I traced back and we never use more than 3 keys for a long time, this clarifies what key would be synced to the URL and helps reduce the usage scope of `DEFAULT_EMBED_DISPLAY_OPTIONS`. Before this, we can only assume all keys in `DEFAULT_EMBED_DISPLAY_OPTIONS` are used, not just `theme` here. So it might be hard to refactor it later on.